packages/client/hmi-client/src/workflow/ops/stratify-mira/stratify-mira-operation.ts

```diff
@@ -0,0 +1,37 @@
+import { Operation, WorkflowOperationTypes } from '@/types/workflow';
+
+export interface StratifyGroup {
+	borderColour: string;
+	name: string;
+	selectedVariables: string[];
+	groupLabels: string;
+	cartesianProduct: boolean;
+}
+
+export interface StratifyOperationStateMira {
+	strataGroups: StratifyGroup[];
+}
+
+export const StratifyMiraOperation: Operation = {
+	name: WorkflowOperationTypes.STRATIFY_MIRA,
+	displayName: 'Stratify MIRA',
+	description: 'Stratify a model',
+	inputs: [{ type: 'modelConfigId', label: 'Model configuration', acceptMultiple: false }],
```

Comment:
Should probably be model-id instead of model-configuration-id

Reply:
I think this should be model-id when our model selection node is changed as well? 
Will be hard to demo if it has a disconnect with all other nodes